docs/se/guides/migration_3.x.adoc

```diff
@@ -0,0 +1,72 @@
+///////////////////////////////////////////////////////////////////////////////
+
```

Comment:
This page also didn't render when I built the PR. Not sure what's wrong.

Reply:
fixed